cl/corpus_importer/management/commands/update_casenames_wl_dataset.py

```diff
@@ -126,6 +126,88 @@ def make_citations(cite1, cite2) -> list[FullCaseCitation]:
     return cites
 
 
+def find_matches(
+    valid_citations: list[FullCaseCitation],
+    csv_docket_num: str,
+    csv_date_filed: date,
+    csv_case_name: str,
+) -> list[tuple[OpinionCluster, str]]:
+    """Find matches for row data
+
+    :param valid_citations: list of FullCaseCitation objects
+    :param csv_docket_num: cleaned docket number from row
+    :param csv_date_filed: formatted filed date from row
+    :param csv_case_name: case name from csv row
+    :return: list of tuples of matched OpinionCluster and used citation
+    """
+    matches = []
+
+    # Try to match row using both citations
+    for citation in valid_citations:
+
+        possible_matches = Citation.objects.filter(
+            **citation.groups,
+            cluster__docket__docket_number__contains=csv_docket_num,
+            cluster__date_filed=csv_date_filed,
+        )
+        if not possible_matches:
+            # Match not found with citation, docket number and date filed
+            continue
+
+        for match in possible_matches:
+            case_name = (
+                match.cluster.case_name_full
+                if match.cluster.case_name_full
+                else match.cluster.case_name
+            )
+            match_on_caption = check_case_names_match(csv_case_name, case_name)
+            if match_on_caption:
+                if not any(
+                    cluster.id == match.cluster.id
+                    for cluster, citation in matches
+                ):
+                    # Avoid duplicates
+                    matches.append(
+                        (match.cluster, citation.corrected_citation())
+                    )
+
+    return matches
+
+
+def update_matched_case_name(
+    matched_cluster: OpinionCluster, csv_case_name: str
+) -> tuple[bool, bool]:
+    """Update case name of matched cluster and related docket
+
+    :param matched_cluster: OpinionCluster object
+    :param csv_case_name: case name from csv row
+    :return: tuple with boolean values if cluster and related docket case name updated
+    """
+    cluster_case_name_updated = False
+    docket_case_name_updated = False
+    if not matched_cluster.case_name or len(csv_case_name) < len(
+        matched_cluster.case_name
+    ):
+        # Save case name in cluster when we don't have it or when the case name in csv is smaller than the current case name
+        matched_cluster.case_name = csv_case_name
+        matched_cluster.save()
+        logger.info(f"Case name updated for cluster id: {matched_cluster.id}")
+        cluster_case_name_updated = True
+
+    if not matched_cluster.docket.case_name or len(csv_case_name) < len(
+        matched_cluster.docket.case_name
+    ):
+        # Save case name in docket when we don't have it or when the case name in csv is smaller than the current case name
+        matched_cluster.docket.case_name = csv_case_name
+        matched_cluster.docket.save()
```

Comment:
drop all this docket name updating.  we should just update the case name for the cluster.